src/goGutter.ts

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import vscode = require('vscode');
+
+export let coveredGutter;
+export let uncoveredGutter;
+
+export function initGutterDecorators(ctx: vscode.ExtensionContext) {
```

Comment:
any reason to have this in a separate function and call from `goMain.ts` instead of doing the same as covered/uncovered Highlight?